tests/integration/test_models/test_full_battery_models/test_lithium_ion/test_dfn.py

```diff
@@ -168,6 +168,10 @@ def positive_radius(x):
 
         param["Negative particle radius [m]"] = negative_radius
         param["Positive particle radius [m]"] = positive_radius
+        # Only get 3dp of accuracy in some tests at 1C with particle distribution
+        # TODO: investigate if there is a bug or some way to improve the
+        # implementation
```

Comment:
is this related to what @brosaplanella was seeing?

Reply:
looks like it. you can get 4dp of accuracy everywhere else.